src/sentry/static/sentry/app/routes.jsx

```diff
@@ -665,6 +665,13 @@ function routes() {
 
   return (
     <Route path="/" component={errorHandler(App)}>
+      <Route
+        path="/accept/:memberId/:token/"
```

Comment:
Should this be a bit more specific? `accept-member` or `accept-org-invite` or `accept-invite` or something?